packaging/windows/build_pypi_pkg/setup.py

```diff
@@ -0,0 +1,64 @@
+import sys
+import subprocess
+
+import setuptools.command.install
+from setuptools import find_packages, setup
+import wheel.bdist_wheel
+
+pkg_name="torch"
+pkg_ver="{{GENERATE_TORCH_PKG_VER}}"
+torch_win_url="https://download.pytorch.org/whl/torch_stable.html"
+
+
+python_min_version = (3, 6, 1)
+python_min_version_str = '.'.join((str(num) for num in python_min_version))
+python_max_version = (3, 9, 0)
+python_max_version_str = '.'.join((str(num) for num in python_max_version))
+
+class install_torch(setuptools.command.install.install):
+    def run(self):
+        setuptools.command.install.install.run(self)
+        raise Exception(
+            f"Can not install torch due to local wheel built failed."
+            f"Plesse visit {torch_win_url} for more details."
+        )
+
+class bdist_wheel(wheel.bdist_wheel.bdist_wheel):
+    def run(self):
+        subprocess.check_call(
+            [sys.executable,'-m', 'pip', 'download', f'{pkg_name}==={pkg_ver}', '-f', torch_win_url, '--no-deps', '-d', self.dist_dir])
```

Comment:
Why download here? Also we should add sth like `--binary-only`, right?

Reply:
Resolved

Reply:
BTW, I still don't understand why do we call `pip download` here.

Reply:
At first, we use pip install here directly, but it will introduce a issue that when user pip uninstall torch, files under caffe2 and torch will not be deleted. The reason is that in RECORD file, it will have file install records for this tar.gz package, without files installed under caffe2 and torch folder. So we changed to pip download, to make pip uninstall torch command fully work as on other platform.